README.md

```diff
@@ -22,9 +22,21 @@ In the specifications in the above table of contents, the keywords "MUST", "MUST
 
 To provide context for users the following section gives example use cases for each part of the spec.
 
-## Filesystem Bundle & Configuration
+#### Application Bundle Builders
 
-- A user can create a root filesystem and configuration, with low-level OS and host specific details, and launch it as a container under an Open Container runtime.
+Application bundle builders can create a [bundle](bundle.md) directory that includes all of the files and some low-level details for launching an application as a container.
+The application can in-turn rely on certain OS configuration being available when it is launched inside of a container by a runtime.
```

Comment:
On Fri, Dec 25, 2015 at 07:46:34AM -0800, Brandon Philips wrote:
> > +Application bundle builders can create a [bundle](bundle.md)
> >  directory that includes all of the files and some low-level
> >  details for launching an application as a container.
> > +The application can in-turn rely on certain OS configuration
> >  being available when it is launched inside of a container by a
> >  runtime.
> 
> OCI is low-level. It points and and in many cases requires things
> like cgroups paths. I think someone familiar with docker image
> format or appc would be surprised by these requirements.

Clearing up that surprise makes more sense to me than the “can in-turn
rely on” phrasing (which doesn't link to docs about what OS
configuration it's referring to).  How about:

  Application bundle builders can create a [bundle](bundle.md)
  directory that includes all of the files required for launching an
  application as a container.  The bundle also contains OCI
  [configuration](config.md) [files](runtime-config.md) where you can
  declare both high-level settings ([which executable to
  launch](config.md#process-configuration)) and low-level settings
  (which [namespaces](runtime-config-linux.md#namespaces) and
  [cgroups](runtime-config-linux.md#control-groups) to join).

Although I still prefer my initial focus on resource limits [1], as it
avoids mentioning the Linux-specific cgroups/namespaces.

[1]: https://github.com/opencontainers/specs/pull/280#issuecomment-166948973